spec/prompt_spec.rb

```diff
@@ -6,44 +6,85 @@
 
 describe Prompt do
   let(:board) { Board.new }
+  subject(:prompt) { described_class.new(board) }
 
-  describe 'Correct Terminal Output' do
-    it 'Should Print the welcome message' do
-      expect { Prompt.welcome }.to output(Message.welcome).to_stdout
+  describe '#welcome' do
+    it 'should Print the welcome message' do
+      expect { prompt.welcome }.to output(Message.welcome).to_stdout
     end
+  end
 
-    it 'Should Print the tic tac toe board' do
-      expect { Prompt.print_board }.to output(board.display_board).to_stdout
+  describe '#print_board' do
+    it 'should Print the tic tac toe board' do
+      expect { prompt.print_board }.to output(board.display_board).to_stdout
     end
+  end
 
-    it 'Should Print the instructions to the game' do
-      expect { Prompt.print_instruction }.to output(
+  describe '#print_instruction' do
+    it 'should Print the instructions to the game' do
+      expect { prompt.print_instruction }.to output(
         Message.instruction
       ).to_stdout
     end
+  end
+
+  describe '#print_invalid_move_error' do
+    it 'should print an error for when a player makes an invalid move.' do
+      expect { prompt.print_invalid_move_error }.to output(
+        Message.invalid_move_error
+      ).to_stdout
+    end
+  end
 
-    it 'Should print the current player' do
+  describe '#print_current_player' do
+    it 'should print the current player' do
       marker = 'X'
-      expect { Prompt.print_current_player(marker) }.to output(
+      expect { prompt.print_current_player(marker) }.to output(
         Message.current_player(marker)
       ).to_stdout
     end
+  end
+
+  describe '#print_tie' do
+    it 'should print that the game ended in a tie' do
+      expect { prompt.print_tie }.to output(Message.tie).to_stdout
+    end
+  end
+  describe '#print_winner' do
+    it 'should print the winner of the game' do
+      marker = 'X'
+      expect { prompt.print_winner(marker) }.to output(
+        Message.winner(marker)
+      ).to_stdout
+    end
+  end
 
-    it 'Should print the current player\'s recent move ' do
+  describe '#print_players_move' do
+    it 'should print the current player\'s recent move' do
       marker = 'X'
-      expect { Prompt.print_players_move(marker, 9) }.to output(
+      expect { prompt.print_players_move(marker, 9) }.to output(
         "\nPlayer X chose the spot labeled: 9\n"
       ).to_stdout
     end
+  end
 
-    it 'Should receive the current player\'s move ' do
-      prompt = Prompt.new
+  describe '#get_players_move' do
+    it 'should receive the current player\'s move ' do
       allow(prompt).to receive(:gets).and_return('9')
       expect(prompt.get_players_move('X')).to eq(9)
     end
+  end
+
+  describe '#get_computers_move' do
+    it 'should receive the current player\'s move ' do
```

Comment:
Do we expect the current player to always be a computer?

Reply:
I should have been more clear! That should say 'should receive the computer's move'.